ui/src/app/components/sidenav-view-assessment/sidenav-view-assessment.component.html

```diff
@@ -1,2 +1,130 @@
+      <mat-icon class="close-icon">close</mat-icon>
+    </button>
+  </div>
+  <div class="content">
+    <p>
+      Estimation for {{ srcDbType.toUpperCase() }} to for Spanner conversion:
+      {{ conversionRatePercentage.good }}% of tables can be converted automatically,
+      {{ conversionRatePercentage.ok }}% requires minimal conversion changes and
+      {{ conversionRatePercentage.bad }}% requires high complexity conversion changes.
+    </p>
+
+    <p></p>
+    <h3>Source and destination database definitions</h3>
+    <table mat-table [dataSource]="dbDataSource" >
+      <ng-container matColumnDef="title">
+        <th mat-header-cell *matHeaderCellDef class="header-2">Title</th>
+        <td mat-cell *matCellDef="let element">{{ element.title }}</td>
+      </ng-container>
+
+      <ng-container matColumnDef="source">
+        <th mat-header-cell *matHeaderCellDef class="header-2">Source</th>
+        <td mat-cell *matCellDef="let element">{{ element.source }}</td>
+      </ng-container>
+
+      <ng-container matColumnDef="destination">
+        <th mat-header-cell *matHeaderCellDef class="header-2">Destination</th>
+        <td mat-cell *matCellDef="let element">{{ element.destination }}</td>
+      </ng-container>
+
+      <tr mat-header-row *matHeaderRowDef="dbDisplayedColumns"></tr>
+      <tr mat-row *matRowDef="let row; columns: dbDisplayedColumns"></tr>
+    </table>
+    <h3>Conversion status by table</h3>
+    <table mat-table [dataSource]="rateCountDataSource" class="conversionByTable">
+      <ng-container matColumnDef="total" >
+        <th mat-header-cell *matHeaderCellDef>Total tables</th>
+        <td mat-cell class="cells" *matCellDef="let element">{{ element.total }}</td>
+      </ng-container>
+
+      <ng-container matColumnDef="bad" class="bad">
+        <th mat-header-cell *matHeaderCellDef>
+          <mat-icon class="icon danger icon-size icons"> error </mat-icon>Converted with many issues
+        </th>
+        <td mat-cell class="cells" *matCellDef="let element">{{ element.bad }}</td>
+      </ng-container>
+
+      <ng-container matColumnDef="ok">
+        <th mat-header-cell *matHeaderCellDef>
+          <mat-icon class="icon warning icon-size icons"> warning</mat-icon>Conversion some warnings &
+          suggestions
+        </th>
+        <td mat-cell  class="cells" *matCellDef="let element">{{ element.ok }}</td>
+      </ng-container>
+      <ng-container matColumnDef="good">
+        <th mat-header-cell *matHeaderCellDef>
+          <mat-icon class="icon success icon-size icons"> check_circle </mat-icon>100% conversion
+        </th>
+        <td mat-cell  class="cells" *matCellDef="let element">{{ element.good }}</td>
+      </ng-container>
+
+      <tr mat-header-row *matHeaderRowDef="rateCountDisplayedColumns"></tr>
+      <tr mat-row *matRowDef="let row; columns: rateCountDisplayedColumns"></tr>
+    </table>
+    <h3>Conversion status by table percentage</h3>
```

Comment:
We don't need this conversion status by percentage , it is not providing any extra information.

Reply:
Updated